pkg/build/api/v1/types.go

```diff
@@ -303,6 +303,10 @@ type BuildStrategy struct {
 
 	// CustomStrategy holds the parameters to the Custom build strategy
 	CustomStrategy *CustomBuildStrategy `json:"customStrategy,omitempty"`
+
+	// JenkinsPipelineStrategy holds the parameters to the Jenkins Pipeline build strategy.
+	// This strategy is experimental.
```

Comment:
How fast can we get an origin image with the plugin?

On Wed, Apr 27, 2016 at 6:46 PM, Ben Parees <notifications@github.com>
wrote:

> In pkg/build/api/v1/types.go
> <https://github.com/openshift/origin/pull/7949#discussion_r61349848>:
>
> > @@ -303,6 +303,10 @@ type BuildStrategy struct {
> >
> >  	// CustomStrategy holds the parameters to the Custom build strategy
> >  	CustomStrategy *CustomBuildStrategy `json:"customStrategy,omitempty"`
> > +
> > +	// JenkinsPipelineStrategy holds the parameters to the Jenkins Pipeline build strategy.
> > +	// This strategy is experimental.
>
> We have a draft of an image, but we need rpm versions of a bunch of
> plugins before we can get a rhel version produced. and i think we're still
> pretty far from the finalized version of the plugin that's needed in the
> jenkins image (in terms of how it gets configured/etc by the user)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7949/files/fb7993e7a126f0d0a611f5d9aa5fad48b35d5e76#r61349848>
>